code/src/components/Form.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+export const Form = ({ messageNew ,setMessageNew, onFormSubmit }) => {
+
+    const onMessageNewChange = (event) => {
+        setMessageNew(event.target.value)
+      }
+
+    return (
+        <>
+            <form className='form-container' onSubmit={onFormSubmit}>
+                <label tabIndex='0' htmlFor='newMessage'>What's making you happy right now?</label>
+                <textarea 
```

Comment:
You could set a maxLength on your textarea to prevent the user from typing more than 140 characters if you wanted to.